apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/entity/condition/ConditionCode.java

```diff
@@ -107,13 +113,33 @@ public class ConditionCode implements Serializable {
     private Character morbReportEnableInd;
 
     @Column(name = "port_req_ind_cd")
-    private Character portReqIndCd;
+    private String portReqIndCd;
 
     @Column(name = "family_cd", length = 256)
     private String familyCd;
 
     @Column(name = "coinfection_grp_cd", length = 20)
     private String coinfectionGrpCd;
 
-
+    @Column(name = "rhap_parse_nbs_ind")
+    private String rhapParseNbsInd;
+
+    @Column(name = "rhap_action_value")
+    private String rhapActionValue;
+
+    @OneToMany(mappedBy = "conditionCd")
+    private Set<LdfPageSet> ldfPageSets = new LinkedHashSet<>();
+
+    public ConditionCode(final ConditionCommand.AddCondition request) {
+        this.id = request.id();
+        this.codeSystemDescTxt = request.codeSystemDescTxt();
+        this.conditionShortNm = request.conditionShortNm();
+        this.nndInd = request.nndInd();
+        this.progAreaCd = request.progAreaCd();
+        this.reportableMorbidityInd = request.reportableMorbidityInd();
+        this.reportableSummaryInd = request.reportableSummaryInd();
+        this.contactTracingEnableInd = request.contactTracingEnableInd();
+        this.familyCd = request.familyCd();
+        this.coinfectionGrpCd = request.coinfectionGrpCd();
+    }
```

Comment:
I believe an entry in the `LDF_page_set` table is created when creating a new condition through legacy. We need to verify that happens in our new API as well